libs/barista-components/core/src/style/_colors.scss

```diff
@@ -1,7 +1,4 @@
 $white: #ffffff;
-/** @breaking-change Flat white will be removed with version 7.0.0 */
-/** @deprecated Use $white instead */
-$flat-white: #ffffff;
```

Comment:
There is a filter in the icon color wheel that we can remove after this one is gone again. Does not need to be in this PR, I just wanted to mention it.

Reply:
Only two lines needed to be removed. I included that in this PR.